.github/workflows/run-tests.yml

```diff
@@ -8,8 +8,44 @@ concurrency:
     group: ${{ github.workflow}}-${{ github.ref }}
     cancel-in-progress: true
 jobs:
+  check-formatting:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: temurin
+          cache: maven
+
+      - name: Check formatting in core
+        run: mvn speedy-spotless:check --batch-mode --no-transfer-progress --update-snapshots -q -f ./dhis-2/pom.xml
+
+      - name: Check formatting in web
+        run:
+          mvn speedy-spotless:check --batch-mode --no-transfer-progress --update-snapshots -q -f ./dhis-2/dhis-web/pom.xml
+  analyze-dependencies:
```

Comment:
I would name it check-dependencies (consistent with check-formatting)

Reply:
Done